### PR TITLE
unit test of delete post

### PR DESCRIPTION
**Pull request by mvn-tamnguyen4-dn**
## Description
unit test delete post : 
+ test click button delete and display confirmation popup
+ test click Close confimation popup
+ test click Ok confirmation popup
+ test with PUT, GET, POST, DELETE method
+ test delete post success and fail
## Reference Requirements
- Product Backlog ID: PBI-11
- Task ID: 5

## Reference Isuees
None
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
None

### CODE REVIEW

**Review by at-phutran2 — COMMENTED** [review]
LGTM

**Comment by at-duyho** [comment]
- you miss case delete fail.

File: tests/Browser/Pages/Backend/Posts/AdminDeletePostTest.php [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+     * @return void
+     */
+    public function testClickButtonDelelePost()
+    {  
+        $post = Post::first();
+        
+        $this->browse(function (Browser $browser) use($post) {
+            $browser->loginAs(User::find(1))
+                    ->visit('/admin/posts/' . $post->id)
+                    ->resize(1500, 1000)
```
Comment (at-phutran2) [inline tests/Browser/Pages/Backend/Posts/AdminDeletePostTest.php]: please apply for all resize ( ->resize(1200, 1600) )

**Review by at-phutran2 — COMMENTED** [review]
LGTM

**Comment by at-duyho** [comment]
- Anh có các issue như thế này:
 + PR này có chức năng là unitest thì chỉ nên làm unittest không nên sửa code để viết unittest.
 + Nếu có bug thì phải tạo một PR hotfix/sprint{num}_{bug}

**Comment by at-duyho** [comment]
- miss case, user don't login. thank!

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**